index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>TIC TAC TOE</title>
+        <link rel="stylesheet" href="tictac.css">
+    </head>
+
+    <body>
+        <div class="container">
+
+            <div class="currentPlayer">
```

Comment:
It's a common practice to use dashes for CSS classes and ids. Let's stick to it. So change the class name to `current-player`.